<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="maximumSize">
       <size>
        <width>784</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabRadon">
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Преобразование Радона&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <attribute name="title">
        <string>Преобразование Радона</string>
       </attribute>
       <widget class="QPushButton" name="radon">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Преобразовать</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="radonNewImage">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>271</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Создать новое изображение</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabRotation">
       <attribute name="title">
        <string>Поворот</string>
       </attribute>
       <widget class="QLabel" name="rotationDegreeLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Угол поворота:</string>
        </property>
       </widget>
       <widget class="QLabel" name="rotationPointLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Точка поворота:</string>
        </property>
       </widget>
       <widget class="QLabel" name="pointCoordsLabel">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>10</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>(null, null)</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="rotationDegreeInput">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>30</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="centerOnScroll">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="rotateButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Повернуть</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="rotateNewImage">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>271</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Создать новое изображение</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="mOpen"/>
    <addaction name="mSave"/>
    <addaction name="separator"/>
    <addaction name="mClose"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="mOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="mSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="mClose">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
